astro/src/content/docs/lifecycle/register-users/advanced-registration-forms.mdx

```diff
@@ -175,23 +176,23 @@ Create a second step. Add your custom house hunting parameter fields:
 
 After you've added these fields to the form, feel free to rearrange the form fields within each step by clicking the arrows to move a field up or down. 
 
-The form configuration specifies steps and field display order within those steps. If you need to move a field between steps, delete it from one step and add it to another. To change field validation, return to the <strong>Fields</strong> section and make your changes. When you're done tweaking the form to your liking, save it.
+The form configuration specifies steps and field display order within those steps. If you need to move a field between steps, delete it from one step and add it to another. To change field validation, return to the **Fields** section and make your changes. When you're done tweaking the form to your liking, save it.
 
 ### Associate a Form With an Application
 
 Once you've created an advanced registration form, the next step is to specify which applications should use this form. Forms can be reused in any application and any tenant.
 
-In addition to specifying the registration form, you'll need to configure a few other options. Assuming you are creating a new FusionAuth application, navigate to the <strong>Applications</strong> tab and add one. If you aren't, you'll need to tweak the settings of your existing application.
+In addition to specifying the registration form, you'll need to configure a few other options. Assuming you are creating a new FusionAuth application, navigate to the **Applications** tab and add one. If you aren't, you'll need to tweak the settings of your existing application.
```

Comment:
I was corrected on this recently. Tabs are still considered navigation and should use `Breadcrumb`. I think all the new uses of `**` in here should be `Breadcrumb`.

Reply:
Fixed